README.md

```diff
@@ -6,20 +6,49 @@ Based on [Outer Wilds Korean Translation](https://outerwildsmods.com/mods/outerw
 
 ## Getting started
 
-- Create a base mod using the regular mod template.
-- Add ILocalizationAPI as an interface in your mod with the following code:
-```cs
-public interface ILocalizationAPI
-{
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath, string assetBundlePath, string fontPath, Func<string, string> fixer);
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath, string assetBundlePath, string fontPath);
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath);
-}
-```
+1. Create a base mod using [the regular mod template.](https://github.com/Raicuparta/ow-mod-template). 
+2. _Optional:_ Rename all files to match the mod name:
```

Comment:
I'd argue this is not optional, because if you compile the mod as "Raicuparta.ModTemplate" it'll become incompatible with any other mod that does the same thing